Signal.xcodeproj/project.pbxproj

```diff
@@ -4431,7 +4439,8 @@
 				PROVISIONING_PROFILE = "";
 				SWIFT_OBJC_BRIDGING_HEADER = "Signal/test/SignalTests-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.0;
+				SWIFT_SWIFT3_OBJC_INFERENCE = On;
```

Comment:
> SWIFT_SWIFT3_OBJC_INFERENCE 

We might want to turn this off? 

> SWIFT_VERSION

Not sure if we should use 4.0 or 4.1.

Reply:
> SWIFT_SWIFT3_OBJC_INFERENCE

has been disabled everywhere.

> SWIFT_VERSION

So, if you select 4.1 in the XCode project/target build settings, the actual value that is persisted is 4.0.  So I think we're okay?